controllers/greptimedbcluster/deployers/datanode.go

```diff
@@ -218,21 +199,6 @@ func (d *DatanodeDeployer) generateSvc(cluster *v1alpha1.GreptimeDBCluster) (*co
 					Protocol: corev1.ProtocolTCP,
 					Port:     cluster.Spec.MySQLServicePort,
 				},
-				{
-					Name:     "postgres",
-					Protocol: corev1.ProtocolTCP,
-					Port:     cluster.Spec.PostgresServicePort,
```

Comment:
question: if deleted, unnecessary define in [type](https://github.com/GreptimeTeam/greptimedb-operator/blob/develop/apis/v1alpha1/greptimedbcluster_types.go#L445) ?

Reply:
The datanode service doesn't need pg or mysql port anymore. It just need HTTP and gRPC service. However, the frontend service still need those settings.